hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -58,15 +65,39 @@ protected boolean requiresRuntime(OperationContext context) {
     @Override
     protected void removeServices(OperationContext context, ServiceName parentService, ModelNode parentModel)
```

Comment:
EAP6.4 seems to be upset with something at this point. I'm trying to reason with it.